lib/rialto/etl/configs/stanford_organizations_to_json_list.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'traject_plus'
+require 'rialto/etl/readers/stanford_organizations_json_reader'
+require 'rialto/etl/writers/json_list_writer'
+
+extend TrajectPlus::Macros
+extend TrajectPlus::Macros::JSON
+
+settings do
+  provide 'writer_class_name', 'Rialto::Etl::Writers::JsonListWriter'
+  provide 'reader_class_name', 'Rialto::Etl::Readers::StanfordOrganizationsJsonReader'
+end
+
+to_field 'url', extract_json('$.url'), single: true
```

Comment:
This is a rather dumb new Traject config that does little mapping; it essentially takes the output of the `StanfordOrganizationsJsonReader` and spits it out.